### ISSUE TITLE
Investigate switching to management boundaries over legislative boundaries

### ISSUE THREAD

**Issue by nateirwin** [nationalparkservice/places-data#118]
This one is likely quite some time off, but IMR brought up that they are going to try to push a management boundary standard through the GIS program. This might be a good alternative for us to switch to - definitely after the initial Park Tiles 3 rollout is complete.

**Comment by chadlawlis** [comment]
Anything new on this end @nateirwin?

**Comment by nateirwin** [comment]
No, this is going to be a long-standing issue. It's really dependent on the NPS GIS community creating a new dataset. I know that Land Resources has a plan to move this forward, but I don't know what the timeframe is.

**Comment by chadlawlis** [comment]
Got it, thanks. Just curious because GOGA's GIS specialist brought up using management instead of legislative boundaries this week, this would be nice if already available.

**Comment by nateirwin** [comment]
Well, it might make sense to prototype this at GOGA. Especially if they have a good dataset.

**Comment by chadlawlis** [comment]
That is what Craig is drawing up for us. It won't be 100% park endorsed as "management boundaries" but very close to it. He's doing his best to conform the management boundaries to what we need for Park Tiles.

**Comment by jakecoolidge** [comment]
Perhaps a heads-up for @nateirwin (I think @cwlawlis802 is aware of this)--we integrated a legislative boundary for the NPNH basemap that incorporates land and water areas.

**Comment by nateirwin** [comment]
Humm, interesting. Any reason we haven't integrated the new boundary back into Park Tiles and/or our Places boundaries?

**Comment by jakecoolidge** [comment]
Related to [this issue](https://github.com/nationalparkservice/places-data/issues/421).
